### PR TITLE
testing/mame: upgrade to 0.210, move to community

### PR DESCRIPTION
**Pull request by tanertas**
This port is in testing for a quite long time. Give a way to the community repo.

### CODE REVIEW

**Comment by tanertas** [comment]
> cd "$builddir" on default functions can be removed

Did it for portmidi. Will do for mame for next update because I don't want to mess up with rebase.

File: community/mame/APKBUILD [hunk 1]
```diff
@@ -48,11 +48,10 @@ subpackages="
 _groovymame_patch=groovymame-$pkgver.diff
 source="
 	https://github.com/mamedev/mame/archive/$pkgname$_pkgver.tar.gz
-	$_groovymame_patch::https://drive.google.com/uc?export=download&id=1_h5lcMQ3xMJlKcrh_u1CtCnQhrQgry7W
+	$_groovymame_patch::https://drive.google.com/uc?export=download&id=1K_JmYATSEwcqQx1JpD4ah42Txlg0H7bc
```
Comment (tcely) [inline community/mame/APKBUILD]: Why is this patch not committed to aports.gir instead of downloaded?

Comment (tanertas) [reply community/mame/APKBUILD]: it is actually a distribution file which gets updated so often.
and is also too big.

**Comment by tanertas** [comment]
I'm closing this.

**Pull request closed without merging**